### PR TITLE
Georgia 2024 - 2029 income tax rates and exemption increase

### LINKED ISSUE #3953: Georgia 2024 - 2029 income tax rates and exemption increase
[Law](https://www.legis.ga.gov/legislation/62346)

### PR DESCRIPTION
**Pull request by laviniawo**
Fixes #3953

### CODE REVIEW

File: policyengine_us/parameters/gov/states/ga/tax/income/agi/exclusions/retirement/cap/earned_income.yaml [hunk 1]
```diff
@@ -15,3 +15,5 @@ metadata:
     href: https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=fb5db531-a80f-4790-bddb-eefc8327ef60&config=00JAA1MDBlYzczZi1lYjFlLTQxMTgtYWE3OS02YTgyOGM2NWJlMDYKAFBvZENhdGFsb2feed0oM9qoQOMCSJFX5qkd&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A65D2-CDH3-CGX8-044N-00008-00&pdcontentcomponentid=234186&pdteaserkey=sr1&pditab=allpods&ecomp=8s65kkk&earg=sr1&prid=66f02b0a-c5ae-4162-9535-127751546807
 values:
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/agi/exclusions/retirement/cap/earned_income.yaml:16]: add the 2023 tax form reference

File: policyengine_us/parameters/gov/states/ga/tax/income/agi/exclusions/retirement/cap/earned_income.yaml [hunk 2]
```diff
@@ -15,3 +15,5 @@ metadata:
...
   2021-01-01: 4_000
+  2023-01-01: 4_000
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/agi/exclusions/retirement/cap/earned_income.yaml:18]:
```suggestion
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ga/tax/income/agi/exclusions/retirement/cap/earned_income.yaml:18]: dont need to add the new year if the value is the same

File: policyengine_us/parameters/gov/states/ga/tax/income/agi/exclusions/retirement/cap/earned_income.yaml (hunk 1 above)
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ga/tax/income/agi/exclusions/retirement/cap/earned_income.yaml:16]: for all yaml files

File: policyengine_us/parameters/gov/states/ga/tax/income/agi/exclusions/retirement/cap/earned_income.yaml (hunk 2 above)
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ga/tax/income/agi/exclusions/retirement/cap/earned_income.yaml:18]: for all yaml files

File: policyengine_us/parameters/gov/states/ga/tax/income/credits/low_income/amount.yaml [hunk 3]
```diff
@@ -38,3 +38,27 @@ brackets:
       2021-01-01: 20_000
     amount:
       2021-01-01: 0
+  - threshold:
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/credits/low_income/amount.yaml:41]: delete if it is the same

File: policyengine_us/parameters/gov/states/ga/tax/income/deductions/standard/amount.yaml [hunk 4]
```diff
@@ -18,6 +18,10 @@ JOINT:
   2021-01-01: 6_000
   2022-01-01: 7_100
   2024-01-01: 24_000
+  #2024-01-01: 18_500
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/deductions/standard/amount.yaml:21]: ?

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_us/parameters/gov/states/ga/tax/income/exemptions/dependent.yaml [hunk 5]
```diff
@@ -14,3 +14,5 @@ metadata:
     href: https://houpl.org/wp-content/uploads/2022/01/2021_IT-511_Individual_Income_Tax_Booklet-12.29.21-Web-Version.pdf#page=13
   - title: Official Code of Georgia Annotated | CH17 Income Taxes | O.C.G.A. § 48-7-26 (b)(3)
     href: https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=5c65c45e-44b1-4c09-be93-e1cf5dc8590c&config=00JAA1MDBlYzczZi1lYjFlLTQxMTgtYWE3OS02YTgyOGM2NWJlMDYKAFBvZENhdGFsb2feed0oM9qoQOMCSJFX5qkd&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A6348-G0H1-DYB7-W3JT-00008-00&pdcontentcomponentid=234186&pdteaserkey=sr0&pditab=allpods&ecomp=7s65kkk&earg=sr0&prid=f512a2f2-3c2b-498a-8f4c-dd1c2ee7934f
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ga/tax/income/exemptions/dependent.yaml:16]: document not available could you update? if the legal code includes these changes we can drop the legislation reference

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

**Pull request merged**